tree/dataframe/src/RLoopManager.cxx

```diff
@@ -360,15 +360,14 @@ RLoopManager::RLoopManager(std::unique_ptr<RDataSource> ds, const ColumnNames_t
 }
 
 RLoopManager::RLoopManager(const ROOT::RDF::RDatasetSpec &spec)
-   : fDefaultColumns(spec.fDefaultColumns), fStartEntry(spec.fStartEntry), fEndEntry(spec.fEndEntry),
-     fNSlots(RDFInternal::GetNSlots()),
+   : fStartEntry(spec.fStartEntry), fEndEntry(spec.fEndEntry), fNSlots(RDFInternal::GetNSlots()),
      fLoopType(ROOT::IsImplicitMTEnabled() ? ELoopType::kROOTFilesMT : ELoopType::kROOTFiles),
      fNewSampleNotifier(fNSlots), fSampleInfos(fNSlots)
 {
    // A TChain has a global name
-   auto chain = std::make_shared<TChain>(spec.fDatasetName.c_str());
+   auto chain = std::make_shared<TChain>(spec.fTreeNames[0].c_str()); // use the first tree name (FOR NOW)
```

Comment:
This should be constructed with `spec.fTreeName`

Reply:
Even better called `spec.fDatasetName` as before

Reply:
Commented above ^ **(https://github.com/vepadulano/root/pull/3#discussion_r892145107)**